### PR TITLE
Priority feature documentation

### LINKED ISSUE #120: Update UG to include Priority feature

### PR DESCRIPTION
**Pull request by Joven-Heng**
close #120

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -51,13 +51,15 @@ Adds a client to I4I.
 A person can have any number of client sources (including 0)
 </div>
 
-Format: `add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [o/NOTE] [s/CLIENTSOURCE]…​`
+Format: `add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [o/NOTE] [s/CLIENTSOURCE]…​? [l/PRIORITY]`
```
Comment (WeiJie96) [inline docs/UserGuide.md:54]: What does the `?` represent?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -81,12 +83,43 @@ Notes:
...
+
+
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ [coming soon]
 
 --------------------------------------------------------------------------------------------------------------------
 
+
+
```
Comment (WeiJie96) [inline docs/UserGuide.md:122]: Perhaps we can update the command summary table?

**Review by WeiJie96 — CHANGES_REQUESTED** [review]
Looks good, just some minor things

File: docs/UserGuide.md (hunk 1 above)
Comment (Joven-Heng) [reply docs/UserGuide.md:54]: Oh my bad

File: docs/UserGuide.md (hunk 2 above)
Comment (Joven-Heng) [reply docs/UserGuide.md:122]: But this is not a command, thats why i didnt update it.

Comment (WeiJie96) [reply docs/UserGuide.md:122]: Oh I mean to update the add command with [l/PRIORITY]

image

**Review by WeiJie96 — CHANGES_REQUESTED** [review]
Some minor things on the command table

**Review by WeiJie96 — APPROVED** [review]
LGTM

**Pull request merged**